.github/workflows/tests.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
         oresat-gen-fw-files c3
         oresat-gen-dcf c3
         oresat-print-od c3 > /dev/null
+        oresat-gen-kaitai
```

Comment:
This checks that the script can run, but I'd also like to see a check that the output can be compiled. That way we can be sure we don't accidentally break the output with changes to our yaml configs. You'd have to install the `ksc` compiler and then `ksc -t python oresat0_5.ksy` or something like that.